.promu.yml

```diff
@@ -0,0 +1,17 @@
+go:
+    cgo: true
+repository:
+    path: github.com/oliver006/redis_exporter
+build:
+    flags: -a -tags 'netgo static_build'
+    ldflags: |
+        -s
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Version={{.Version}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Revision={{.Revision}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.Branch={{.Branch}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.BuildUser={{user}}@{{host}}
+        -X {{repoPath}}/vendor/github.com/prometheus/common/version.BuildDate={{date "20060102-15:04:05"}}
+tarball:
+    files:
+        - LICENSE
+        - NOTICE
```

Comment:
Can you also add the cross builds - i think that will come in handy with the automated circleCI builds:

```
crossbuild:
    platforms:
        - linux/amd64
        - linux/386
        - darwin/amd64
        - darwin/386
        - netbsd/amd64
        - netbsd/386
        - linux/arm
        - linux/arm64
```